README.md

```diff
@@ -34,104 +34,55 @@ Getting Started
 
 Ketrew is very flexible and hence may seem difficult to understand and setup at
 first.
-Let's get a minimal workflow running.
+Let's get a minimal setup ready and a workflow running on it.
 
+### Server-side Setup
 
+We use 
+[`hammerlab/secotrec`](https://github.com/hammerlab/secotrec),
+to get a practical Ketrew/Coclobas local setup.
 
+For this we'll require Docker installed (usable by a regular UNIX user, and its
+daemon running), to get a container environment with everything installed:
 
+    docker pull hammerlab/secotrec
+    docker run -v /var/run/docker.sock:/var/run/docker.sock -it hammerlab/secotrec bash
```

Comment:
ah right, that shouldn't work on docker-for-mac or docker-for-windows …

Discussed offline, I'm going to ditch the whole `docker run` thing and ask people to install Secotrec, Docker, and Docker-compose.